webapp/components/test-runs-query.html

```diff
@@ -205,10 +211,17 @@
           this.setProperties(batchUpdate);
         }
 
-        computeResultsRangeMessage(from, maxCount) {
+        computeResultsRangeMessage(from, to, maxCount) {
           if (from) {
-            var diff = Math.floor((Date.now() - from) / 86400000);
-            return `Showing test runs from the last ${diff} days`;
+            const max = to || Date.now();
```

Comment:
`max` is only used to compute `diff` and diff is only used if `to` is falsy, so the purpose of this line is hard to parse. Seems like it could be replaced with `const max = Date.now()`. Maybe `if (to) { /* compute and return stuff */ } else { /* other stuff */ }` to make this bit more clear?

Reply:
Done.